template.go

```diff
@@ -14,8 +14,12 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/spf13/afero"
+	"github.com/zealic/xignore"
 )
 
+// Gomplateignore ignore file name, like .gitignore
+const Gomplateignore = ".gomplateignore"
```

Comment:
Let's leave this unexported for now - I don't see a reason to export this yet:

```suggestion
const gomplateignore = ".gomplateignore"
```

(obviously references below will need to change)